### PR TITLE
Have SheetValues return a Promise

### PR DESCRIPTION
**Pull request by mattmarcin**
Makes life easier when first loading Data

```
SheetValues.new("SHEETID", "0"):andThen(function(SheetManager)
	print(SheetManager)
end)
```

### CODE REVIEW

File: wally.toml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 [package]
-name = "boatbomber/sheetvalues"
-description = "Use Google Sheets as a live value updater for Roblox games."
-version = "3.0.2"
+name = "mattmarcin/sheetvalues"
```
Comment (boatbomber) [inline wally.toml:2]: This is not your package, not to mention I literally couldn't publish this.

File: wally.toml [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+description = "Use Google Sheets as a live value updater for Roblox games. Returns a Promise"
```
Comment (boatbomber) [inline wally.toml:3]: Package descriptions are not the place for API documentation.

File: wally.toml [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+version = "3.0.4"
```
Comment (boatbomber) [inline wally.toml:4]: Completely changing the return contract is not a patch version bump. Not to mention that you skipped from 2 to 4.

File: wally.toml [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 license = "MIT"
 authors = ["boatbomber (https://boatbomber.com)"]
 registry = "https://github.com/upliftgames/wally-index"
 realm = "server"
 include = ["src", "src/**", "wally.toml", "wally.lock", "default.project.json"]
 exclude = ["**"]
+
+[dependencies]
+promise = "evaera/promise@4.0.0"
```
Comment (boatbomber) [inline wally.toml:13]: I prefer capitalized package names, and an eof blank line.

File: default.project.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "SheetValues",
-	"tree": {
-		"$path": "src"
-	}
+  "name": "sheetvalues",
```
Comment (boatbomber) [inline default.project.json:2]: Please don't change the name of the outputted Instances.

**Review by boatbomber — CHANGES_REQUESTED** [review]

File: wally.toml (hunk 1 above)
Comment (mattmarcin) [reply wally.toml:2]: Sorry about that. I published it for my own use but didn't realize it also went to this PR after the fact. I'll fix this up,

**Pull request closed without merging**